site/app/models/Config.php

```diff
@@ -167,6 +167,12 @@ class Config extends AbstractModel {
      * @var string */
     protected $cgi_tmp_path;
 
+    /**
+     * Indicates whether Submitty is running in CI (to mock certain 'random' functions)
+     * @var bool
+     * @prop
+     */
+    protected $ci = false;
```

Comment:
Isn't this the same as `$debug` in practice?

Reply:
I guess it is, from reading the phpdoc about debugging, I assumed that it was mainly for development environments to output full exceptions, but I guess we can use it as a 'ci mode' as well..